benchmarking/benchmark.py

```diff
@@ -293,6 +323,7 @@ def main():
         pprint(results)
         with open(config['results_path'], 'wt') as f:
             json.dump(results, f)
+        push_result_s3(config['results_path'])
```

Comment:
Should uploading to s3 be optional? Someone might want to run this benchmarking locally and just see the output as before?

Reply:
:+1: 
I'll just check if the environment variables have been set, not adding an extra one.